src/pudl/metadata/fields.py

```diff
@@ -449,6 +449,22 @@
         ),
         "unit": "min",
     },
+    "can_cofire_100_oil": {
+        "type": "boolean",
+        "description": "Can the generator co-fire 100 oil?",
+    },
+    "can_cofire_oil_and_gas": {
+        "type": "boolean",
+        "description": "Can the generator co-fire oil and gas?",
+    },
+    "can_fuel_switch": {
+        "type": "boolean",
+        "description": "TK",
+    },
+    "can_switch_when_operating": {
+        "type": "boolean",
+        "description": "Indicates whether the generator switches fuel while operating.",
+    },
```

Comment:
I like the way you formatted the second description better than the first one. I.e., use "Indicates whether...." rather than framing the definition as a question.